src/main/java/com/coscale/sdk/client/ApiClient.java

```diff
@@ -421,7 +423,12 @@ public String getRequesUrl(String endpoint, Options options, boolean globalApi)
         }
         sb.append(endpoint);
 
-        if (options != null) {
+        if (options != null && options.hasQuery()) {
+            // No ? yet in the string, then add one.
+            if (sb.indexOf("?") != -1) {
```

Comment:
Only if the ? is not in the URL yet, add it. And that only if the options instance is not null and has something to add to the query.